### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by prageethw** [asobti/kube-monkey#116]
the values.yaml

```yaml
replicaCount: 1
namespace: default
rbac:
  enabled: true
image:
  repository: ayushsobti/kube-monkey
  tag: v0.2.3
  pullPolicy: IfNotPresent
config:
  dryRun: false
  runHour: 8
  startHour: 10
  endHour: 16
  blacklistedNamespaces: kube-system
  whitelistedNamespaces:
  timeZone: Australia/Melbourne
  debug:
   enabled: true # if you want to enable debugging and see how pods killed immediately set this to true
   schedule_immediate_kill: true
args:
  logLevel: 5
  logDir: /var/log/kube-monkey
```
ms values.yaml
```yaml
kubemonkey:
  enabled: enabled # to disable set this to "disabled", to enable set it to "enabled"
  mtbf: 1
  kill-mode: fixed-percent
  kill-value: 80
```

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xe28cd2]

goroutine 21 [running]:
github.com/asobti/kube-monkey/chaos.(*Chaos).terminate(0xc42015da10, 0x114ba00, 0xc4200b23c0, 0x0, 0x0)
	/Users/asobti/Coding/go/src/github.com/asobti/kube-monkey/chaos/chaos.go:127 +0x1a2
github.com/asobti/kube-monkey/chaos.(*Chaos).Execute(0xc42015da10, 0xc420084660)
	/Users/asobti/Coding/go/src/github.com/asobti/kube-monkey/chaos/chaos.go:66 +0x202
github.com/asobti/kube-monkey/chaos.(*Chaos).Schedule(0xc42015da10, 0xc420084660)
	/Users/asobti/Coding/go/src/github.com/asobti/kube-monkey/chaos/chaos.go:42 +0x51
created by github.com/asobti/kube-monkey/kubemonkey.ScheduleTerminations
	/Users/asobti/Coding/go/src/github.com/asobti/kube-monkey/kubemonkey/kubemonkey.go:56 +0xb1

**Comment by asobti** [comment]
I'll take a look at this in #103

**Comment by asobti** [comment]
@prageethw Can you pull down #103 and see if you still have this issue? I think it should be fixed

**Comment by asobti** [comment]
@prageethw I pushed a new image `kube-monkey:PR103`

**Comment by prageethw** [comment]
@asobti looks good please close the defect tested in all the modes...

**Comment by asobti** [comment]
Awesome. Thanks @prageethw

**Comment by asobti** [comment]
Fixed by #103
